packages/salesforcedx-vscode-replay-debugger/src/states/variableAssignmentState.ts

```diff
@@ -29,7 +29,10 @@ export class VariableAssignmentState implements DebugLogState {
         name.indexOf('.') > -1 ? name.substring(0, name.lastIndexOf('.')) : '';
       const varName =
         nameSplit.length > 0 ? nameSplit[nameSplit.length - 1] : name;
-      const value = this.fields[4];
+      let value = this.fields[4].replace('(^")|("$)', "'");
+      if (value.startsWith('"')) {
```

Comment:
If the value is just primitive then we see if it starts with a double quote and swap it for single quotes.

Reply:
Can this if statement be true if the double quote is replaced by the line before this?

Reply:
Oh I added the if check because the replace regex wasn't working. I am going to remove the replace call.

Reply:
You can probably do this `replace(/^"(.*)"$/, "'$1'")`

Reply:
Going to use `.replace(/^"/, "'").replace(/"$/, "'")`  as we discussed since the `$` syntax is not a standard feature
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/n